packages/components/src/public/QueryModel/SaveViewModal.tsx

```diff
@@ -13,6 +13,7 @@ import { isProductProjectsEnabled } from '../../internal/app/utils';
 import { useServerContext } from '../../internal/components/base/ServerContext';
 
 const MAX_VIEW_NAME_LENGTH = 200;
+const RESERVED_VIEW_NAMES = ['default', 'my default', '~~details~~', '~~insert~~', '~~update~~'];
```

Comment:
I planned to switch our sample finder views to ~~samplefinder~~ from 'Sample Finder' at some point but forgot about it.. Maybe this is a good time to make the switch, and add ~~samplefinder~~ to the set of reserved views.

Reply:
Done.  Should be safe enough since these are session views that will go away when the new code is deployed.